frontend/src/arrivalFlareEffects.js

```diff
@@ -1,10 +1,10 @@
 // Get location of element and create two circle elements at that location with css effects
-const arrivalEffectCreate = (elementId, arrivalEffects) => {
+const arrivalFlareEffectCreate = (elementId, arrivalFlareEffects) => {
```

Comment:
nice, improved variable naming is where I get my kicks these days and I do approve of this !